docs/api/authenticate.md

```diff
@@ -3,4 +3,134 @@ id: authenticate
 title: How to Authenticate with our API
 ---
 
-This chapter will talk about the authentication process.
\ No newline at end of file
+If you are entering the API for the first time as a not logged in user, you will have access to the objects available only publicly. For example, by default, you can query public data, such as published products or pages.
+
+To fetch protected data, like orders or users, you need to authenticate your access.
+Saleor API uses a [JWT token](https://jwt.io/) authentication mechanism.
+Once you create a token, you need to include it as a header with every GraphQL request.
+
+The authorization header has the following format:
+
+```
+Authorization: JWT token
+```
+
```

Comment:
```suggestion

> **Note**
> You need to leave the `JWT` keyword before the token in the header value. Otherwise it will not work.

```

Reply:
There are at least three examples, I think it should be clear.